openTSNE/tsne.py

```diff
@@ -1593,6 +1616,8 @@ def __call__(
         momentum=0.8,
         exaggeration=None,
         dof=1,
+        optimize_for_alpha=False,
+        dof_lr=0.5,
```

Comment:
Can we connect this to the t-SNE learning rate somehow? Don't like the extra parameter